Tests/scripts/hook_validations/tests/integration_test.py

```diff
@@ -701,3 +701,156 @@ def test_is_default_arguments_ok():
 
     assert validator.is_default_arguments() is True, \
         "The integration validator found an invalid command arg while it is valid"
+
+
+def test_is_outputs_for_reputations_commands_valid():
+    validator = IntegrationValidator("temp_file", check_git=False)
+    validator.current_integration = {
+        "script": {
+            "commands": [
+                {
+                    "name": "panorama-commit-status",
+                    "outputs": [
+                        {
+                            "contextPath": "Panorama.Commit.JobID",
+                            "description": "Job ID of the configuration to be committed",
+                            "type": "number"
+                        },
+                        {
+                            "contextPath": "DBotScore.does.not.matter"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+    validator.old_integration = None
+
+    assert validator.is_outputs_for_reputations_commands_valid() is True, \
+        "The integration validator found invalid command outputs while it is valid"
+
+    validator_email = IntegrationValidator("temp_file", check_git=False)
+    validator_email.current_integration = {
+        "script": {
+            "commands": [
+                {
+                    "name": "email",
+                    "outputs": [
+                        {
+                            "contextPath": "DBotScore.Indicator",
+                            "description": "The indicator that was tested.",
+                            "type": "string"
+                        },
+                        {
+                            "contextPath": "DBotScore.Type",
+                            "description": "The indicator type.",
+                            "type": "string"
+                        },
+                        {
+                            "contextPath": "DBotScore.Vendor",
+                            "description": "Vendor used to calculate the score.",
+                            "type": "string"
+                        },
+                        {
+                            "contextPath": "DBotScore.Sc0re",
```

Comment:
```suggestion
                            "contextPath": "DBotScore.Score",
```

Reply:
@yaakovi it should be sc0re
the assert is False